config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits.yaml

```diff
@@ -50,3 +50,17 @@ presubmits:
         - "--scenario=execute"
         - "--"
         - "./scripts/ci-test.sh"
+  - name: pull-cluster-api-vendor-in-sync
+    always_run: true
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/cluster-api:v20181020-5ce477da2
```

Comment:
v20190114-652973a54 now :-)

Reply:
https://github.com/kubernetes/test-infra/pull/9874#issuecomment-454143375

Reply:
Thank you!